src/recon_buildblock/SPECTUB_Tools.cxx

```diff
@@ -30,6 +36,216 @@ namespace SPECTUB
 #define abs(a) ((a) >= 0 ? (a) : (-a))
 #define SIGN(a) (a < -EPSILON ? -1 : (a > EPSILON ? 1 : 0))
 
+#define DELIMITER1 '#' // delimiter character in input parameter text file
```

Comment:
All of this was removed in 9a85f5471524051a3463698823bdc1a1cacfe485. Either you need to merge master on here, or you did some copy-paste.